resources/metadata/mtaBuild.yaml

```diff
@@ -48,6 +48,22 @@ spec:
           - STEPS
         mandatory: false
         default:
+      - name: source
+        type: string
+        description: "The path to the MTA project."
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: false
+      - name: target
+        type: string
+        description: "The folder for the generated `MTAR` file. If the parameter has been provided, the `MTAR` file is saved in the root of the folder provided by the argument."
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: false
```

Comment:
I would add the empty default: parameter as well as it is done consistently in that file